data/src/main/kotlin/land/sungbin/androidprojecttemplate/data/BuyerReview.kt

```diff
@@ -0,0 +1,54 @@
+package land.sungbin.androidprojecttemplate.data
+
+/**
+ * 구매자 거래후기[BuyerReview] data class
+ *
+ * @param buyerId 구매자 id
+ * @param sellerId 판매자 id
+ * @param isDirect 거래방식 (= 직거래인지, 아니라면 택배거래)
+ * @param saleState [SaleState] 거래 간단 상태
+ * @param likeReason [LikeReason] 좋은 점
+ * @param dislikeReason [DislikeReason] 아쉬운 점
+ * @param etc 기타 (string)
+ */
+data class BuyerReview(
+    val buyerId: String,
+    val sellerId: String,
+    val isDirect: Boolean,
+    // TODO (riflockle7) serializer 로 int 와 대응하도록 처리 필요
+    val saleState: SaleState,
+    // TODO (riflockle7) serializer 로 int 와 대응하도록 처리 필요
+    val likeReason: LikeReason,
+    // TODO (riflockle7) serializer 로 int 와 대응하도록 처리 필요
+    val dislikeReason: DislikeReason,
+    val etc: String,
+)
+
+/** 거래 간단 상태 */
+enum class SaleState {
+    Sorry,  // 아쉬워요
+    Like,   // 좋아요
+    Great,  // 최고예요
```

Comment:
```suggestion
    Great; // 최고예요
```

enum 은 마지막에 `;` 으로 끝내는게 어떤가용
몬가 이게 좀 더 정석같은 느낌

Reply:
함수가 있을때는 저게 필수이긴한데, 해당 enum class 는 함수가 없는 상태여서요...!
오히려 함수가 없다는 걸 인식하기엔 반점(`,`) 이 더 좋은 것 같습니다

Reply:
`함수가 있을때는 저게 필수이긴한데` 아하!! 몰랐네요 감사해요 
그럼 지금처럼 유지하는게 좋겠네요!